examples/sf/example.js

```diff
@@ -281,8 +281,8 @@ export default class Example extends Component<
         }
       ]
     });
-    const nebulaLayers = [segmentsLayer, editableJunctionsLayer, textLayer];
-    const deckLayers = [editablePolygonsLayer];
+    const nebulaLayers = [segmentsLayer, editableJunctionsLayer];
```

Comment:
Maybe separate the examples that use `nebulaLayers`.

Reply:
Agreed. We'll have some cleanup to do with the docs and examples after things solidify.